app/js/components/manageApps/ManageApps.jsx

```diff
@@ -193,7 +197,7 @@ export default class ManageApps extends React.Component {
               }));
 
         }).then(() => {
-          const installedAddons = installedOwas.concat(installedModules);
```

Comment:
Is this change part of the ticket?

Reply:
The proper implementation had been removed in a previous PR. I was just putting it back.